generators/app/prompts.js

```diff
@@ -51,12 +51,12 @@ async function askForApplicationType() {
             name: 'Monolithic application (recommended for simple projects)',
         },
         {
-            value: 'microservice',
-            name: 'Microservice application',
+            value: 'gateway',
```

Comment:
Why putting gateway before microservice?

Reply:
easier to understand IMO
- to not confuse between Microservice application and Microservice gateway
- the 1st and 2nd option have front
- the 3rd and 4th backend only

Reply:
ok, make sens now.

Reply:
it's because I select the wrong option several times... :-)